woocommerce/woocommerce
# Add 'template_name' labels to Product Category and Product Tag taxonomies

### Changes proposed in this Pull Request:

As https://github.com/WordPress/gutenberg/pull/60367 and https://github.com/WordPress/wordpress-develop/pull/6344 have been merged/approved, we can now use `template_name` labels for WooCommerce custom post types and custom taxonomies.

Closes https://github.com/woocommerce/woocommerce/issues/47933.

### How to test the changes in this Pull Request:

0. Make sure you have at least one product category and one product tag in your store. You can go to Products > Categories and Products > Tag to create one.
1. With the Gutenberg 18.5.0 or later, go to Appearance > Editor > Templates > Add New Template.
2. Verify WooCommerce templates have user-friendly names. Those are: _Products by Category_ and _Products by Tag_.

Before | After
--- | ---
![image](https://github.com/woocommerce/woocommerce/assets/3616980/6dd9f918-deb1-4ccb-8b60-9857e9ec3da9) | ![image](https://github.com/woocommerce/woocommerce/assets/3616980/5c9ce918-b605-4d64-a7ec-f69df6f602ee)


### Changelog entry







<details>

#### Significance




#### Type




#### Message 


</details>

<details>

#### Comment 


</details>
